recipes/libsimpleitk/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "libsimpleitk" %}
+{% set version = "1.2.0" %}
+{% set sha256 7edac57f7a146163746b76e8bdeb9bf8b3cf3633c4e8e7ea3291121240d69d82
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SimpleITK/SimpleITK/releases/download/v{{version}}/SimpleITK-{{version}}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+    number: 0
+    # Lets first get Linux and OSX working...
+    skip: True # [win and py27]
+    features:
+    - vc9  # [win and py27]
+    - vc14  # [win and py>=35]
+
+requirements:
+  build:
+    - cmake   >=3.10
+    - ninja
+    - lua     5.1.*
+    - libitk  4.13.*
+    - python  # [win]
+    - vc 9  # [win and py27]
+    - vc 14  # [win and py35>=35]
```

Comment:
I don't think this vc stuff is needed anymore, that is what compiler('c') and 'cxx' do.